tests/test_api.py

```
@@ -6,47 +6,65 @@
     # Using TF2's ID as it's a well-established game on GB
-    game_profile = api.get_game_profile(297)
+    game_profile = await api.get_game(297)
     assert game_profile is not None
     assert hasattr(game_profile, "name")
     assert game_profile.preview_media is not None
 
-def test_api_error(api):
+
+@pytest.mark.asyncio
+async def test_api_error(api: PyBanana):
     with pytest.raises(Exception):
```

Ruff (0.12.2)

Do not assert blind exception: `Exception`

(B017)